src/autoscaler/mta.tpl.yaml

```diff
@@ -9,41 +9,61 @@ build-parameters:
   before-all:
     - builder: custom
       commands:
-      - make clean vendor
+      - make clean vendor vendor-changelogs clean-dbtasks package-dbtasks
+
 
 modules:
-  - name: metricsforwarder
+  - name: dbtasks
+    type: java
+    path: dbtasks/target/db-1.0-SNAPSHOT.jar
+    properties:
+      JBP_LOG_LEVEL:
+      DEBUG:
+    build-parameters:
+      builder: custom
+      commands: []
+    requires:
+    - name: database
+    parameters:
+      no-route: true
+      no-start: true
+      tasks:
+      - name: apply-api-changelog
+        command: " chmod +x ./BOOT-INF/classes/bin/apply-api-changelog.sh ; /home/vcap/app/BOOT-INF/classes/bin/apply-api-changelog.sh"
```

Comment:
In future, we can turn this bash script in java probably (testable and less bash )